bitnami/zookeeper/templates/prometheusrules.yaml

```
@@ -0,0 +1,22 @@
+{{- if and .Values.metrics.enabled .Values.metrics.prometheusRule.enabled .Values.metrics.prometheusRule.rules }}
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: {{ include "zookeeper.fullname" . }}
+  {{- if .Values.metrics.prometheusRule.namespace }}
+  namespace: {{ .Values.metrics.prometheusRule.namespace }}
+  {{- else }}
+  namespace: {{ .Release.Namespace }}
+  {{- end }}
+  labels: {{- include "zookeeper.labels" . | nindent 4 }}
+    app.kubernetes.io/component: zookeeper
+    {{- range $key, $value := .Values.metrics.prometheusRule.selector }}
+    {{ $key }}: {{ $value | quote }}
+    {{- end }}
+spec:
+  groups:
+  - name: {{ include "zookeeper.fullname" . }}
+    rules:
+    {{- toYaml .Values.metrics.prometheusRule.rules | nindent 6 }}
```

Shouldn't it be like this? (to avoid that extra newline)

```suggestion
    rules: {{- toYaml .Values.metrics.prometheusRule.rules | nindent 6 }}
```

---

Interestingly it doesn't actually make any difference to the output. I'll commit that change though. Looks a bit tidier.